lib/article_json/import/google_doc/html/node_analyzer.rb

```diff
@@ -105,6 +113,19 @@ def type
             return :embed if embed?
             :unknown
           end
+
+          private
+
+          # Return true if the node only contains <br> nodes and empty text
+          # @return [Boolean]
+          def only_includes_brs?
+            return false unless node.inner_text.strip.empty?
+            tags = node.children.map(&:name)
+            # Check if it only contains <br> and text nodes
+            return false unless tags.all? { |tag| %w(br text).include? tag }
+            # Check if at least one is a `<br>` node
+            tags.any? { |tag| tag == 'br' }
```

Comment:
I think `tags.include?('br')` would be nicer.

Reply:
Good point – I'll change it right away :)

Reply:
Changed :tada: